packages/angular/cli/models/command-runner.ts

```diff
@@ -70,18 +101,27 @@ export async function runCommand(commandMap: CommandMap,
     ? CommandScope.inProject
     : CommandScope.outsideProject;
 
-  let Cmd: CommandConstructor | null;
-  Cmd = commandName ? findCommand(commandMap, commandName) : null;
+  const commandMapPath = findUp('commands.json', __dirname);
```

Comment:
Possible to make the config file path as a parameter?

Reply:
Done.